client/src/pages/RoomPage/index.tsx

```diff
@@ -210,7 +210,7 @@ function RoomPage({ name }: { name: string }): JSX.Element {
       stream.getAudioTracks()[0].enabled = !selfUserInfoRef.current.mute;
     }
     networkRef.current?.updateInfo(selfUserInfoRef.current);
-  }, [selfUserInfoRef.current]);
+  }, [selfUserInfoRef.current.mute, stream]);
```

Comment:
This is being used to update all info not just mute so this dependency is incorrect.

Reply:
It's probably better to split each hook for readability